tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -354,6 +356,112 @@ IncludeByKey.PASSTHROUGH {
 			}
 		}
 
+		IncludeByKey.NUM_DMICS {
+			"[1-4]" {
+						IncludeByKey.ASRC_CAPTURE_ENABLE {
```

Comment:
indentation ?

Reply:
Do I read this correctly, we have?
```
IncludeByKey.PASSTHROUGH {
"false" {
	...
	IncludeByKey.NUM_DMICS {
	"[1-4]"	{
		...
		}
	}

+	IncludeByKey.NUM_DMICS {
+	"[1-4]" {
+		IncludeByKey.ASRC_CAPTURE_ENABLE {
+		"true"	{
+			...
+			}
+		}
+	}
	...
```

Is there some reason for having two NUM_DMIC [1-4] blocks following each other, or is this use of big nested IncludebyKey blocks just getting out of hand?

Not in the scope of this PR, but it could be a time to split nocodec file into smaller pieces included from within IncludeByKey blocks.